components/reactLazyLoad.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { translate } from '../helpers/l10n';
+import { requestTryAndRepeatUntil } from '../helpers/request';
+import { Alert } from './ui/Alert';
+
+interface ImportError {
+  request?: string;
+}
+
+export function reactLazyLoad<T extends React.ComponentType<any>>(
```

Comment:
How about `lazyLoadComponent`, I'm weirdly bothered by `reactLazyLoad`

Reply:
Yeah I didn't put too much thought into the name, started with `newLazyLoad` xD
I'm fine with `lazyLoadComponent` :+1: